cmake/modules/RootMacros.cmake

```diff
@@ -891,7 +926,7 @@ function(ROOT_ADD_INCLUDE_DIRECTORIES library)
       # get a redefinition error.
       # We should remove these lines when the fallback include is removed. Then
       # we will need a module.modulemap file per `inc` directory.
-      target_include_directories(${library} BEFORE PRIVATE ${CMAKE_BINARY_DIR}/include)
+      target_include_directories(${library} PRIVATE ${CMAKE_BINARY_DIR}/include)
```

Comment:
Please explain the reasoning for this change. I don't understand why add this directory if it's at the end of the list.